exonum/src/runtime/rust/tests.rs

```diff
@@ -199,7 +199,7 @@ impl<T: Runtime> Runtime for Inspected<T> {
         instance_id: u32,
     ) -> Result<(), ExecutionError> {
         let schema = CoreSchema::new(&*context.fork);
-        // Skip genesis block execution.
+        // Skip adding events during genesis block execution.
```

Comment:
Sorry that didn't point out this before, but checking whether `after_transactions` is run twice on the genesis block could be useful. Would it be difficult to refactor this method to record `schema.next_height()` instead of `schema.height()`?

Reply:
Yeah, that looks like a good idea. As an addition, I made **all** the hooks use `next_height`, because
- It's better to have the same timescale everywhere;
- It allowed us to replace `Option<Height>` with `Height` for `CommitService` event.